code/controllers/subsystem/tickets/tickets.dm

```diff
@@ -107,12 +108,16 @@ SUBSYSTEM_DEF(tickets)
 	var/datum/ticket/T = allTickets[N]
 	if(T.ticketState != TICKET_RESOLVED)
 		T.ticketState = TICKET_RESOLVED
+		message_staff("<span class='[span_class]'>[usr.client] / ([usr]) resolved [ticket_name] number [N]</span>")
```

Comment:
Staff messaging and ticket closing messages were moved to the procs that handle their opening, closing and resolving.